packages/swapper/src/types.ts

```diff
@@ -422,13 +427,22 @@ export type CheckTradeStatusInput = {
   config: SwapperConfig
 } & EvmSwapperDeps &
   UtxoSwapperDeps &
-  CosmosSdkSwapperDeps
+  CosmosSdkSwapperDeps &
+  SolanaSwapperDeps
 
 // a result containing all routes that were successfully generated, or an error in the case where
 // no routes could be generated
 type TradeQuoteResult = Result<TradeQuote[], SwapErrorRight>
 export type TradeRateResult = Result<TradeRate[], SwapErrorRight>
 
+export type SolanaTransactionRequest = {
+  gasLimit: string
+  to: string
+  from: string
+  value: string
+  data: string
+}
+
```

Comment:
Remove me, unused and not a Solana request

Reply:
Damn, removed in [4e76e81](https://github.com/shapeshift/web/pull/8120/commits/4e76e8115223d91bd602f7d9ceed80c94c77ea15)